### PR TITLE
NO-JIRA Fix build-wrapper download URL

### PR DESCRIPTION
**Pull request by 7PH**
Fixes the obsolete URL to download the build wrapper

### CODE REVIEW

File: scripts/configure_paths.sh [hunk 1]
```diff
@@ -65,7 +65,7 @@ echo "sonar-scanner-dir=${SONAR_SCANNER_DIR}"
 echo "sonar-scanner-bin=${SONAR_SCANNER_DIR}/bin/${SONAR_SCANNER_NAME}"
 
 BUILD_WRAPPER_DIR="${INSTALL_PATH}/build-wrapper-${BUILD_WRAPPER_SUFFIX}"
-echo "build-wrapper-url=${SONAR_HOST_URL%/}/static/cpp/build-wrapper-${BUILD_WRAPPER_SUFFIX}.zip"
+echo "build-wrapper-url=${SONAR_HOST_URL%/}/sonarqube/static/cpp/build-wrapper-${BUILD_WRAPPER_SUFFIX}.zip"
```
[the remark this answers is not in the document]
Comment (7PH) [reply scripts/configure_paths.sh:68]: No, you are correct, I didn't see that `configure_path.sh` was used in the action code and not the tests. I don't think it should be modified so I've reverted this, thanks for spotting 👍

File: .github/workflows/tests.yml [hunk 2]
```diff
@@ -110,7 +110,7 @@ jobs:
     runs-on: ubuntu-latest
     env:
       INSTALL_PATH: 'install-directory'
-      SONAR_HOST_URL: 'http://sonar-host.com'
+      SONAR_HOST_URL: 'http://sonar-host.com/sonarqube/'
```
Comment (7PH) [inline .github/workflows/tests.yml:113]: added a trailing slash as it's supposed to be removed by `configure_paths.sh`

File: scripts/configure_paths.sh (hunk 1 above)
[the remark this answers is not in the document]
Comment (michael-jabbour-sonarsource) [reply scripts/configure_paths.sh]: Thanks for following with the failing tests!

**Review by michael-jabbour-sonarsource — APPROVED** [review]
I added one comment because I expected the required changeset to be smaller.

I don't have a strong opinion about it, so feel free to merge. The only argument I could think of is that the URL is expected to be shorter for most users, so I would prefer to keep internal tests without it...

File: .github/workflows/tests.yml (hunk 2 above)
Comment (7PH) [reply .github/workflows/tests.yml]: Yes, I agree.. thanks for your due diligence 😀

**Pull request merged**